DoSomething/gladiator
# Adds contest API

#### What's this PR do?
- Added new endpoint to get a contest based on run nid
- Added new transformer for the unique response returned by ^
- Added new function to waiting room model to determine if it should be open for signups or not

#### How should this be manually tested?
Hit the endpoint http://homestead.app/api/v1/contests?run_nid=<run_nid>
Make sure the response looks good, something like this

![screen shot 2016-03-11 at 11 31 37 am](https://cloud.githubusercontent.com/assets/897368/13708476/d6401470-e77c-11e5-84b7-b44dbc36dd53.png)

Play around with changing the waiting room dates as well to make sure the signups_open works as intended

#### Any background context you want to provide?
See conversation in #Laravel for why I made a new transformer 

#### What are the relevant tickets?
Fixes #63
